cleanlab_studio/internal/util.py

```diff
@@ -258,10 +258,16 @@ def tracked_func(*args: Any, **kwargs: Any) -> Any:
                     if track_all_frames:
                         cleanlab_traceback = trace_str
                     else:
+                        # remove stack frames for user code
                         cleanlab_match = re.search("File.*cleanlab", trace_str)
                         cleanlab_traceback = (
                             trace_str[cleanlab_match.start() :] if cleanlab_match else ""
                         )
+                        # then remove the user file path from the stack trace
+                        pattern = re.compile(r".*?(File).*?(cleanlab-studio.*)")
```

Comment:
This pattern will miss removing user file paths from some lines if the error occurs in another library called by `cleanlab-studio`